<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калибровка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="lstPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сохранить значение калибровочных коэффициентов в энергонезависимую память&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Загрузить значение калибровочных коэффициентов из энергонезависимой памяти&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>760</width>
      <height>130</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabOut">
     <attribute name="title">
      <string>Выход</string>
     </attribute>
     <widget class="QComboBox" name="lstOutGain">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpModeOut">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>251</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Режим</string>
      </property>
      <widget class="QRadioButton" name="rbOutMin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Минимум</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbOutMax">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Максимум</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="OutMax">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>мВ</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>14000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="OutMin">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>мВ</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>14000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QDoubleSpinBox" name="outCal">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>мВ</string>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>12750.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFixOut">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Зафиксировать промежуточное значение для указанного режима"/>
      </property>
      <property name="text">
       <string>Зафиксировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalOut">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Внести калибровочные коэффициенты для указанного диапазона"/>
      </property>
      <property name="text">
       <string>Калибровать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Частота:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Frequency">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>Гц</string>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>12000</number>
      </property>
      <property name="value">
       <number>400</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Усиление:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="outCalPhase">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>˚</string>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QToolButton" name="btnCoefOut">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIn">
     <attribute name="title">
      <string>Вход</string>
     </attribute>
     <widget class="QComboBox" name="lstInGain">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpChannelIn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Канал</string>
      </property>
      <widget class="QRadioButton" name="rbIn1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbIn2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btnCoefIn1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>22</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btnCoefIn2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>22</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpModeIn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>251</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Режим</string>
      </property>
      <widget class="QRadioButton" name="rbInMin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Минимум</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbInMax">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Максимум</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="inMax">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>мВ</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>14000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="inMin">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>мВ</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>14000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnCalIn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Внести калибровочные коэффициенты для указанного диапазона"/>
      </property>
      <property name="text">
       <string>Калибровать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFixIn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Зафиксировать промежуточное значение для указанного режима"/>
      </property>
      <property name="text">
       <string>Зафиксировать</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="inCal">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>мВ</string>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>12750.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Усиление:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="inCalPhase">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>˚</string>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRef">
     <attribute name="title">
      <string>Опорный</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabCurrent">
     <attribute name="title">
      <string>Ток</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="textLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>761</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
